integ-test-promote-template.yml

```diff
@@ -456,3 +459,4 @@ jobs:
             grep "deploying" buildlog.txt
             rm buildlog.txt
           displayName: Verify that correct tasks appear in buildlog.txt
+          condition: not(eq(${{ version }}, '0'))
```

Comment:
This is a problem with the template approach. I think we'll need to solve it before the feature diffs between the versions continues to grow larger.

Reply:
What about having a common template and v1 and v0 (if necessary) templates? Then you can stick the conditional at the top-level to decide what version specific template is included.